pkg/addonmanager/controllers/agentdeploy/utils.go

```diff
@@ -491,7 +491,7 @@ func getManifestConfigOption(agentAddon agent.AgentAddon,
 
 		manifests, err := agentAddon.Manifests(cluster, addon)
 		if err != nil {
-			return manifestConfigs
+			return manifestConfigs, fmt.Errorf("get all deployments error: %v", err)
```

Comment:
Will the GetValues error propagate here, right?

Reply:
yes